src/scripts/types/image-type.ts

```diff
@@ -0,0 +1,4 @@
+export enum ImageType {
+  Photo = 'photo',
+  Chart = 'chart'
```

Comment:
Chart is probably not the best type here. It could be any other image which should be contained. Why not call it what it is like 'contain' and 'cover'?

Reply:
Sure, i can change it. I just thought contain and cover would be a bit confusing in the story mapper.

Reply:
Yes good point, but I guess labeling a photograph as  "chart" just to display it as contained is more confusing. At least for me 😄